apisix/core/ctx.lua

```diff
@@ -213,6 +213,10 @@ do
         route_name = true,
         service_id = true,
         service_name = true,
+        resp_body = function(ctx)
+            -- only work when file-logger set include_resp_body = true
```

Comment:
The comment is incorrect. See https://github.com/apache/apisix/pull/8711#discussion_r1089879862

Reply:
done